### DISCUSSION TITLE
Q:  Migrate comments from WordPress site || Q: Migrate comments from WordPress site

### DISCUSSION THREAD

**Discussion by wuhan005** [walinejs/waline#2348] [Q&A] [answered]
### Describe the bug | 问题描述

The migration helper (https://waline.js.org/en/migration/tool.html) doesn't provide a way to migrate the comments data from a WordPress site.

Are there any other tools or best practices to do this?

### Website URL | 问题网站

N/A

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

MongoDB

---
### Describe the bug | Problem description

The migration helper (https://waline.js.org/en/migration/tool.html) doesn't provide a way to migrate the comments data from a WordPress site.

Are there any other tools or best practices to do this?

### Website URL | Problem website

N/A

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

MongoDB

**Comment by lizheming** [comment, marked as answer]
可以先迁移到 Disqus 再转换到 Waline

---
You can migrate to Disqus first and then convert to Waline
